spec/factories/messages.rb

```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
+  factory :message do
+    text 'MyText'
```

Comment:
Maybe you are already using Faker, if not it's a good moment to get in touch with that gem